app/views/hearings/_attendees.html.haml

```diff
@@ -1,28 +1,57 @@
 %div{ class: "govuk-!-margin-bottom-9" }
   %h2.govuk-heading-l
     = t('hearings.show.attendees.heading')
+
+  -# TODO: refactor: add method is_an_appeal? in the decorator
+  - is_an_appeal = hearing_details && hearing_details.hearing_type.description.include?('Appeal')
```

Comment:
I don't think you should be searching for specific words in strings here. For one thing, this will immediately break when we start adding breaches etc and court applications that aren't appeals. For another, the text in the description is only indirectly related to the problem you're trying to solve.

The problem you have is that sometimes you'll be on this screen and `hearing_details.prosecution_cases` will be blank, in which case you need to populate this screen from appeal details rather than prosecution case details. So your check should be more like `is_an_appeal = hearing_details.prosecution_cases.blank?`